models/DataObject/AbstractObject.php

```diff
@@ -1359,8 +1359,10 @@ public static function __callStatic($method, $arguments)
         $db = \Pimcore\Db::get();
 
         if (in_array(strtolower($realPropertyName), self::$objectColumns)) {
-            $arguments = array_pad($arguments, 4, 0);
-            [$value, $limit, $offset, $objectTypes] = $arguments;
+            $value = $arguments[0] ?? null;
```

Comment:
Should we throw an exception for required arguments instead of setting it to `null`?

Reply:
Like that, added the exceptions